CHANGELOG.md

```diff
@@ -5,13 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Version [3.4.2]
+
+## Refactor
```

Comment:
Prima verder maar is refactor hier de juiste benaming? Je verhoogt hem met een minor dan zou je geen refactor verwachten :p

Reply:
Ja lastig, in mijn ogen een refactor van een bestaande validatie. Uitbreiding kan je het ook noemen. Wellicht een 'fix'?

Reply:
Ik heb er fix van gemaakt, aanpassing ook gedaan in de commit message.